src/main/webapp/translate_sentiment.html

```diff
@@ -19,8 +19,19 @@
         <div class="form">
             <h2>Translated Sentence Analyzer</h2>
             <img class="AIimg" src="https://4.bp.blogspot.com/-DdXsUqajhXg/WzC9YkIovmI/AAAAAAABM4M/MIMXmbaCCvwUqhU48c-aphnp9bNNPrICACLcBGAs/s800/book_gijutsusyo_ai.png">
-            <p>Type/Paste your text below:</p>
-        <form method="POST" action="/translate_sentiment.html">
+        <form style="padding: 1%;" method="POST" action="/translate_sentiment.html">
+            <p style="display: inline;">Choose a Language: </p>
+            <div class="dropdown">
+                <select id="chose-langauge" class="drop-button">Language
```

Comment:
nit: choose-language

Reply:
Ohhh I am a silly goose haha I'll change that now